corehq/apps/integration/static/integration/js/hmac_callout.js

```diff
@@ -34,11 +29,11 @@ hqDefine("integration/js/hmac_callout", ["hqwebapp/js/initial_page_data"], funct
 
         var variables = url.searchParams.toString();
 
-        var hashedBody = hash(variables);
+        var hashedBody = hash("variables=" + variables);
 
-        var dest = anchor.href.split('?')[0];
+        var dest = url.origin + "/" + url.pathname;
```

Comment:
I think this "/" is redundant, and will make the resultant url

`https://origin.com//downstream/path`

Which some servers are fine with, but plenty aren't, unless this is something specific to polyfill

Reply:
```suggestion
        var dest = url.origin + url.pathname;
```

Reply:
Good catch; I was doing my testing in the console and apparently my font size is too small for my old eyes. Fix in next push.